code/modules/mob/living/simple_animal/friendly/lizard.dm

```diff
@@ -24,13 +24,66 @@
 	can_collar = TRUE
 	mob_biotypes = MOB_ORGANIC | MOB_BEAST | MOB_REPTILE
 	gold_core_spawnable = FRIENDLY_SPAWN
+	var/eats_bugs = TRUE
+	var/eating_sound = 'sound/weapons/bite.ogg'
+	/// Lizards start with a tail
+	var/still_has_tail = TRUE 
```

Comment:
Not required but I would recommend just `has_tail`.

Reply:
I didn't end up doing "has_tail" because there's a define that uses that and I didn't want to cause confusion, still want it changed?

Reply:
I think it's distinct enough, due to the other being in caps, to have it be `has_tail`

Reply:
Changed to "has_tail"